outcomes/firefox_desktop/newtab_visibility.toml

```diff
@@ -0,0 +1,58 @@
+friendly_name = "New Tab visibility"
+description = """
+    Measures if New Tab is enabled for new tabs and new windows.
+    If these measures decline, users are taking some action that disables New Tab.
+"""
+
+[metrics.new_tab_enabled_on_new_tabs]
+friendly_name = "New Tab enabled on new tabs"
+description = "Whether New Tab is enabled for new tabs in existing windows."
+select_expression = """
+    ARRAY_AGG(
+        mozfun.map.get_key(environment.settings.user_prefs, "browser.newtabpage.enabled") IS NULL
+        ORDER BY submission_timestamp DESC
+        LIMIT 1
+    )[SAFE_OFFSET(0)]
+"""
+data_source = "main"
```

Comment:
Should I worry that `main` is an expensive data source? We might have aggregated this somewhere, somehow.

Reply:
`main` is also seems the reason for exceeding the complexity budget here. Replacing `main` with `main_1pct` runs without any problems

Reply:
I was just playing around with the query a bit. If we query directly from `telemetry_stable.main_v4` everything runs without any problems. I'm guessing having the `telemetry.main` view is adding just a little too much complexity here.